megamek/src/megamek/client/ui/swing/MegaMekGUI.java

```diff
@@ -69,13 +72,15 @@
 import static megamek.common.Compute.d6;
 
 public class MegaMekGUI implements IPreferenceChangeListener {
-    private static final String FILENAME_MEGAMEK_SPLASH = "../misc/megamek-splash.jpg";
+    private static final String FILENAME_MEGAMEK_SPLASH = "../misc/megamek_splash_spooky_hd.png";// "../misc/megamek-splash.jpg";
     private static final String FILENAME_ICON_16X16 = "megamek-icon-16x16.png";
     private static final String FILENAME_ICON_32X32 = "megamek-icon-32x32.png";
     private static final String FILENAME_ICON_48X48 = "megamek-icon-48x48.png";
     private static final String FILENAME_ICON_256X256 = "megamek-icon-256x256.png";
 
     private static final String FILENAME_BT_CLASSIC_FONT = "btclassic/BTLogo_old.ttf";
+    private static final int DEFAULT_DISPLAY_DPI = 96;
+
```

Comment:
```suggestion
    private static final int DEFAULT_DISPLAY_DPI = 96;
```

Reply:
not sure what the diff is here - I did change the line spacing in my most recent commit, maybe that is what you meant

Reply:
It removes the additional empty line, but that change works too.